mac

```diff
@@ -190,6 +190,14 @@ if [ "$brew_last_change" -le "1471233600" ]; then
   print_done
 fi
 
+# Fix permission issue on /usr/local on OS 10.12
+if [ ! -w /usr/local ]; then
+  print_status "Fixing /usr/local permissions issue"
+  sudo chgrp -R staff /usr/local
```

Comment:
wfm. i felt `chgrp staff` was less intrusive, but i think the assumption that these are single-user laptops is reasonable.